src/sidebar/templates/login-control.html

```diff
@@ -35,7 +31,7 @@
     <li class="dropdown-menu__row">
       <a class="dropdown-menu__link" ng-click="vm.onShowHelpPanel()">Help</a>
     </li>
-    <li class="dropdown-menu__row" ng-if="vm.auth.status === 'logged-in' && !vm.isThirdPartyUser()">
+    <li class="dropdown-menu__row" ng-if="vm.auth.status === 'logged-in'">
```

Comment:
Yes, although up to you whether to do it now since all the publishers we are currently intending to integrate with have some kind of logout option.